bootstrap.sh

```diff
@@ -93,7 +93,7 @@ if [ "$automake_sub_version" -lt "15" ]; then
 
     autotools_repos=(http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz
         http://ftp.gnu.org/gnu/automake/automake-1.15.tar.gz
-        http://mirror.team-cymru.org/gnu/libtool/libtool-2.4.2.tar.gz
+        http://mirror.team-cymru.org/gnu/libtool/libtool-2.4.6.tar.gz
```

Comment:
I had to bump libtool up a level because margo seemed to require it.  To get that to work, I had to wipe out my automake install to force this code to rerun.

More autotools fun...

I'm wondering if we shouldn't just have bootstrap always build a particular set of autotools.  That would ensure that everyone building through bootstrap is using a consistent set of tools.

@CamStan , are there problems if disable the automake version check so that this always runs?

Reply:
I don't think there'd be any problems disabling the version check. I only did that to avoid installing additional things if not needed.

FYI: I was able to get bootstrap to work using an install of Autotools in a shared location on our systems as well. I updated that one to use libtool v2.4.6 per this change.